docs/reference/indices/apis/freeze.asciidoc

```diff
@@ -6,6 +6,8 @@
 <titleabbrev>Freeze index</titleabbrev>
 ++++
 
+deprecated::[8.0, Frozen indices are no longer useful due to https://www.elastic.co/blog/significantly-decrease-your-elasticsearch-heap-memory-usage[recent improvements in heap memory utilization]]
```

Comment:
Two nits: I think the deprecation release is actually 7.14. I'd also use `usage` instead of `utilization`.
```suggestion
deprecated::[7.14, Frozen indices are no longer useful due to https://www.elastic.co/blog/significantly-decrease-your-elasticsearch-heap-memory-usage[recent improvements in heap memory usage]]
```